### ISSUE TITLE
Feature Request - Analog read of pins

### ISSUE THREAD

**Issue by plains203** [OXRS-IO/OXRS-IO-StateGPIO-ESP32-FW#3]
Would be a nice feature to also have the ability to read analog pin voltages.

**Comment by sumnerboy12** [comment]
Agreed, this would be pretty handy. Also lends itself towards needing pin level config.

**Comment by sumnerboy12** [comment]
After creating https://github.com/OXRS-IO/OXRS-IO-DigitalIO-ESP32-FW I am of the opinion analog handling should be in separate firmware - there is a fair bit of stuff you could do with analog inputs/outputs so I think it warrants its own code.
